src/test/groovy/pl/tomaszdziurko/jvm_bloggers/mailing/BlogPostForMailItemSpec.groovy

```diff
@@ -41,7 +41,7 @@ class BlogPostForMailItemSpec extends Specification {
             blogPostForMailItem.authorLabel == "<a href=\"https://twitter.com/" + twitter.substring(1) + "\">" + name + "</a>"
     }
 
-    def "Should store author name as a label if twitter account is not set correctly."() {
+    def "Should store author name as a label if twitter account is set with empty string"() {
```

Comment:
_Should store author name as a label if twitter account is set with empty string *or null string*_